topic-service/src/main/java/com/information/center/topicservice/model/request/TopicRequest.java

```diff
@@ -9,6 +9,6 @@
 @NoArgsConstructor
 public class TopicRequest {
 
-  private String name;
+    private String name;
```

Comment:
here you just format the code

Reply:
I'm using a tool for code format

Reply:
if you don't change anything in the class is no need to reformat